### DISCUSSION TITLE
Update 20230917: Firefox installed

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1425] [General]
During the update to 20230917 firefox was downloaded and installed (firefox for a headless system? Consequence of using WebSockets?)

**Comment by ceejayemm** [comment]
Replaces Chromium for local display (ie RPi Official 7in display) - see #1411.  I use this display and Firefox fixes issues that are not (?) going to be fixed in Chromium any time soon. Whether it should be downloaded for headless installation I can't say but Rern seems to replace all files with every update.

Chris

**Comment by klassikmann** [comment]
Thank you Chris. This makes sense.
